### PR TITLE
update usage message

### PR DESCRIPTION
**Pull request by yiwen101**
To be consistent with usage in UG.
All the parameter value the user need to input is now in upper case, values in bracket like [value] indicates optional values and not in bracket indicates necessary values.

### CODE REVIEW

**Comment by yiwen101** [comment]
Fixes #263

File: src/main/java/seedu/address/logic/parser/LinkCommandParser.java [hunk 1]
```diff
@@ -73,22 +73,22 @@ private LinkCommand staticParse(String args) throws ParseException {
     }
     public String getStaticUsageInfo() {
         return "Link command usage: link "
-                + "-student [STUDENT_NAME]"
-                + "-lesson [LESSON_NAME]"
+                + "-student STUDENT_NAME"
+                + "-lesson LESSON_NAME"
```
Comment (pzl111) [inline src/main/java/seedu/address/logic/parser/LinkCommandParser.java]: i think this part is fine, just maybe need to specify in UG that STUDENT_NAME and LESSON_NAME follow the same constraints as NAME parameter

**Review by pzl111 — APPROVED** [review]
just a minor note to remember when updating UG in future

File: src/main/java/seedu/address/logic/parser/LinkCommandParser.java (hunk 1 above)
Comment (yiwen101) [reply src/main/java/seedu/address/logic/parser/LinkCommandParser.java]: Emm, sure

**Pull request merged**